src/FakeItEasy/FakeItEasy.nuspec

```diff
@@ -13,10 +13,41 @@
     <licenseUrl>https://github.com/FakeItEasy/FakeItEasy/blob/master/License.txt</licenseUrl>
     <copyright>Copyright (c) FakeItEasy contributors. (fakeiteasyfx@gmail.com)</copyright>
     <tags>TDD unittesting mocks mocking fakes faking stubs stubbing spy spies doubles isolation substitutes substitution</tags>
+    <dependencies>
+      <group targetFramework=".NETFramework4.0">
+        <dependency id="Newtonsoft.Json" version="9.0.1" />

```

Comment:
We've been dependency-free in the past. I think depending on the System and Microsoft packges makes sense, but wonder if we can't avoid depending on Newtonsoft.Json.

Currently we merge Castle.Core (except for the .NETStandard1.6 case). Perhaps we could do the same for Newtonsoft.Json.
Or even better, could we avoid using Newtonsoft.Json altogether for the .NETFramework4.* builds?